README.md

```diff
@@ -10,10 +10,13 @@ Documentation
 
 See [reference](http://developer.lightbend.com/docs/alpakka/0.2/) and [API](http://developer.lightbend.com/docs/api/alpakka/0.2/akka/stream/alpakka/index.html) documentation pages.
 
-Contributions
--------------
+Contributions & Maintainers
+---------------------------
+
+*Alpakka does not have contributors, it only has maintainers—frequent and infrequent—and everyone helps out.*
```

Comment:
I added `The Akka core team keeps an eye on the project to assure its overall coherence but does not fully support these modules.` though not sure if it's good, please reword at will.